tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -546,4 +546,63 @@ spec:
 			helper.CmdShouldPass("odo", "service", "delete", "EtcdCluster/etcd", "-f")
 		})
 	})
+
+	Context("When linking devfile component with another component", func() {
```

Comment:
If you want to apply the format described in https://github.com/openshift/odo/pull/4770, you can write something similar to:

- When "deploying two components"
    - when "linking one component to another"
        - it "should list the link with the specific name"

- When "deploying one component"
    - it "should fail linking to itself"

Reply:
Updated to use a similar structure.